src/index.ts

```diff
@@ -52,6 +52,19 @@ const plugin: JupyterFrontEndPlugin<void> = {
   ) => {
     console.log('JupyterLab extension jupyterlab_run_and_reload is activated!');
 
+    let saveBeforeRun = true;
+    let saveAfterRun = true;
+
+    function loadSetting(setting: ISettingRegistry.ISettings): void {
+      // Read the settings and convert to the correct type
+      saveBeforeRun = setting.get('saveBeforeRun').composite as boolean;
```

Comment:
why is type assertion needed if the settings have an interface?

Reply:
Made ticket #13 for this to be added later.